StarVMC/StarAgmlLib/AgPlacement.h

```diff
@@ -36,63 +36,53 @@ class AgPlacement : public TNamed, public AgParameterList<double>
 
   TGeoCombiTrans *matrix();
 
-  //  const Char_t *block();
-  //  const Char_t *mother();
-  //  const Char_t *group();
-
-  /// Returns a reference to the named parameter.
-  //  Double_t &par( const Char_t *name );
-
-  /// Tests whether the named parameter is set for this shape
-  //  Bool_t isSet( const Char_t *par ) const;  
-
   enum { kOnly=0, kMany };
 
  private:
  protected:
 
   bool mHasRotm;
 
-  //  std::map< TString, Double_t > mParameters;
-
-  //  TString mBlock;
-  //  TString mMother;
-  //  TString mGroup;
+public:
 
   struct Rotation3 
   {
-    Double_t alpha;
-    Int_t    axis;
+    double alpha;
+    int    axis;
   };
+
   struct Rotation6
   {
-    Double_t thetax, phix, thetay, phiy, thetaz, phiz;
+    double thetax, phix, thetay, phiy, thetaz, phiz;
   };
 
-//  union Rotation	//!
-  struct Rotation	//!
+  union Rotation
   {
     Rotation3 rot3;
     Rotation6 rot6;
-    Short_t   ort[3];
+    short     ort[3];
   };
 
   struct Translation
   {
-    Double_t x;
-    Double_t y;
-    Double_t z;
+    double x;
+    double y;
+    double z;
     Translation() : x(0), y(0), z(0){ };
   };
 
   enum { kUnknown, kRot3, kRot6, kRotO };
   enum { kAlphaX,  kAlphaY, kAlphaZ };
 
+  //#ifndef __CINT__
   Translation             mTranslation;
-  std::vector< Rotation > mRotation;		//!
-  std::vector< Int_t    > mType;		//!
+#if  !defined(__CLING__)
+  std::vector< Rotation > mRotation;
+#endif
```

Comment:
What is the reason to hide this from rootcling? Did you mean rootcint?

Reply:
No, I did mean cling here.  I have a vague recollection that cling was giving me an error on that particular line of code, and so I hid it.

Reply:
Addressed in 1bc5b0c4fd923ab1fdcd2c55ad600c42ae411f28